src/sagemaker/jumpstart/curated_hub/types.py

```diff
@@ -0,0 +1,40 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module stores types related to SageMaker JumpStart."""
+from __future__ import absolute_import
+from typing import Dict, Any
+
+from sagemaker.jumpstart.types import JumpStartDataHolderType
+
+
+class HubContentDocument_v2(JumpStartDataHolderType):
```

Comment:
consider making a separate module for hub content document schemas, as there may be many

Reply:
True, this is a placeholder though while @jinyoung-lim works on the implementation

Reply:
Consider also renaming `HubContentDocumentV2`